lib/schematype.js

```diff
@@ -825,7 +825,7 @@ SchemaType.prototype.get = function(fn) {
  * From the examples above, you may have noticed that error messages support
  * basic templating. There are a few other template keywords besides `{PATH}`
  * and `{VALUE}` too. To find out more, details are available
- * [here](#error_messages_MongooseError-messages).
+ * [here](/docs/api/error.html#Error.prototype.name).
```

Comment:
We make links relative in the docs now, not sure exactly which PR that was but @hasezoey worked on it. So I think this should be `[here](./error.html#Error.prototype.name)` . But please test and confirm.

Reply:
works

Reply:
actually we mostly currently use absolute paths in the jsdoc, for example:

https://github.com/Automattic/mongoose/blob/master/lib/index.js#L931-L938

i wanted to change this at some point so that the urls are valid for both in-editor use and on the website (via using full paths with `https://mongoosejs.com/` and changing the paths in the website processing)